forwarder/update_handlers/tdct_serialiser.py

```diff
@@ -79,11 +79,14 @@ def serialise(
             raise RuntimeError(
                 f'Unable to extract TDC data from EPICS type: "{update.getID()}"'
             )
-        try:
-            if update.value.size == 0:
-                return None, None
-        except AttributeError:
-            pass
+        if isinstance(update.value, type(None)):
```

Comment:
Does `if update.value is None:` not work?

Reply:
Or `if not update.value`?

Reply:
Sounds good! And "if update.value is None:" seems to work

Reply:
Then 'if not update.value' should work too and it is a bit more pythonic

Reply:
Since update.value can be either an array, integer or None, the tests did not succeed with 'if not update.value'  because of the "ambiguity" if an array comes.

Could be solved with any(), but I thought it would look cleaner as it is now